Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -12,28 +12,32 @@ public enum ActorStates
 
 public class ActorPhysics : ActorComponent
 {
+	private new Transform transform;
```

Comment:
How would the implementation for lazy initialized this work? Wouldn't you need another variable anyway?